kf_lib_data_ingest/etl/transform/guided.py

```diff
@@ -0,0 +1,31 @@
+"""
+Module for transforming source data into target service entities via
+a user supplied transform function which specifies how the source data tables
+should be merged in order to yield a single table per target service entity.
+"""
+import logging
+
+from kf_lib_data_ingest.etl.configuration.transform_module import (
+    TransformModule
+)
+
+
+class GuidedTransformer():
```

Comment:
I still don't understand why GuidedTransformer and AutoTransformer are separate classes instead of one class that optionally uses an external module to override some or all of its run method during init.